utils/spacewalk-common-channels.ini

```diff
@@ -1379,13 +1426,60 @@ checksum = sha256
 base_channels = debian-10-pool-amd64
 repo_url = http://security-cdn.debian.org/debian-security/dists/buster/updates/main/binary-amd64/
 
+[debian-10-amd64-client-master]
```

Comment:
Mmmmm, why do we have this and the next channel, if we already have `debian-10-amd64-uyuni-client-stable` and `debian-10-amd64-uyuni-client-master` below?

Also, despite it was not changed by this PR, can we change `-master` to `-devel` as we have for all other OS, and remove `-stable` as we have for all other OS?

Reply:
> why do we have this and the next channel, if we already have debian-10-amd64-uyuni-client-stable and debian-10-amd64-uyuni-client-master below?

because i assumed we want to have these repo duplicated without ```-uyuni``` suffix for later SUMA support?

> Also, despite it was not changed by this PR, can we change -master to -devel as we have for all other OS, and remove -stable as we have for all other OS

.

Yeah fine.
I have noticed something else, we need to add ```for Uyuni``` also in the  ```name``` line otherwise ```spacewalk-common-channels``` complain it exists.

Reply:
> because i assumed we want to have these repo duplicated without -uyuni suffix for later SUMA support?

Not really. SUSE Manager would have its own client tools on SCC, without using `spacewalk-common-channels' :-)

Reply:
gotcha!

let me wrap up and send over
doc updated already

Reply:
done the changes. let me know if it's ok